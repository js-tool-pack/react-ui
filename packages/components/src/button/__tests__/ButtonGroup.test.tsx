import { render } from '@testing-library/react';
import { ButtonGroup, Button } from '..';

describe('ButtonGroup', () => {
  test('size', () => {
    expect(
      render(
        <ButtonGroup size="small">
          <Button></Button>
          <Button></Button>
        </ButtonGroup>,
      ).container.firstChild,
    ).toMatchSnapshot();

    expect(
      render(
        <ButtonGroup size="large">
          <Button></Button>
          <Button></Button>
        </ButtonGroup>,
      ).container.firstChild,
    ).toMatchSnapshot();
  });
});
