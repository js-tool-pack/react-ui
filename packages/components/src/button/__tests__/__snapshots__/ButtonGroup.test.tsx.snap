// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup size 1`] = `
<div
  class="t-button-group"
  role="group"
>
  <button
    class="t-button t-button--type-default t-button--icon-l t--size-sm"
  />
  <button
    class="t-button t-button--type-default t-button--icon-l t--size-sm"
  />
</div>
`;

exports[`ButtonGroup size 2`] = `
<div
  class="t-button-group"
  role="group"
>
  <button
    class="t-button t-button--type-default t-button--icon-l t--size-lg"
  />
  <button
    class="t-button t-button--type-default t-button--icon-l t--size-lg"
  />
</div>
`;
